netutils/constants.py

```diff
@@ -15,6 +15,7 @@
     "Bd": "Bdi",
     "Bdi": "Bdi",
     "Bridge-Aggregation": "Port-channel",
+    "Di": "Dialer",
```

Comment:
```suggestion
    "Dialer": "Dialer",
    "Di": "Dialer",
```

We always map the forward back to itself as well.